### ISSUE TITLE
Deeplinks not working with facebook plugin

### ISSUE THREAD

**Issue by aggarwalankush** [driftyco/ionic-plugin-deeplinks#102]
I'm using http://ionicframework.com/docs/native/facebook/ plugin for facebook login. This plugin is blocking ionic deeplinks as mentioned here - https://github.com/jeduan/cordova-plugin-facebook4/issues/166

**Comment by claudiozam** [comment]
Hi, I have the same problem but only in IOS. Android works ok.

```
Cordova CLI: 6.5.0 
Ionic Framework Version: 3.1.0
Ionic CLI Version: 2.2.1
Ionic App Lib Version: 2.2.0
Ionic App Scripts Version: 1.3.4
ios-deploy version: 1.9.0 
ios-sim version: 5.0.12 
OS: macOS Sierra
Node Version: v7.6.0
Xcode version: Xcode 8.3.2 Build version 8E2002 
```

**Comment by aggarwalankush** [comment]
Yes, in android it works for me too. Seems like some issue with safariviewcontroller. If I use inappbrowser to open system safari, it works perfectly but If I use in app safariviewcontroller, it doesn't work.

**Comment by scottcarter87** [comment]
One observation I have made is that if you try to launch a deep link before using the facebook plugin at all for logging in they work fine.  As soon as you attempt to login once the deep links will no longer work.

**Comment by peterpeterparker** [comment]
I kind of face that issue too... 

If I do a Facebook login (using the cordova-plugin-facebook4 plugin) it will return a `nomatch` route which I could use to proceed the login. 

Later on (if I don't restart the app), if I try to use a deeplink, nothing will be catched by the ionic-plugin-deeplinks plugin but if I check the Xcode console, I could see that somewhere in the Facebook plugin, the link was catched (I found `FB handle url` in the console output).

P.S.: Maybe linked with following issues too? 

https://github.com/ionic-team/ionic-plugin-deeplinks/issues/122
https://github.com/ionic-team/ionic-plugin-deeplinks/issues/105
